addons/point_of_sale/static/src/js/models.js

```diff
@@ -1709,7 +1709,7 @@ class Orderline extends PosModel {
      *    @param {Object} modifiedPackLotLines key-value pair of String (the cid) & String (the new lot_name)
      *    @param {Array} newPackLotLines array of { lot_name: String }
      */
-    setPackLotLines({ modifiedPackLotLines, newPackLotLines }) {
+    setPackLotLines({ modifiedPackLotLines, newPackLotLines }, setQuantity = true) {
```

Comment:
put the new arg inside the first param. It's better than risking stability of the method.

```suggestion
    setPackLotLines({ modifiedPackLotLines, newPackLotLines, setQuantity = true }) {
```